tests/helpers/start-app.js

```diff
@@ -3,14 +3,14 @@ import Application from '../../app';
 import Router from '../../router';
 import config from '../../config/environment';
 
-export default function startApp(attrs) {
-  var application;
+export default function startApp( attrs ) {
+  let application;
 
-  var attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = Ember.merge( {}, config.APP );
+  attributes = Ember.merge( attributes, attrs ); // use defaults, but you can override;
 
-  Ember.run(function() {
-    application = Application.create(attributes);
+  Ember.run( function() {
+    application = Application.create( attributes );
     application.setupForTesting();
     application.injectTestHelpers();
   });
```

Comment:
The rest of this file should following spacing and formatting standards. Change one change them all :-)